linux_os/guide/system/selinux/package_policycoreutils-python-utils_installed/rule.yml

```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Install policycoreutils-python-utils package'
+
+description: |-
+    {{{ describe_package_install(package="policycoreutils-python-utils") }}}
+
+rationale: |-
+    Security-enhanced Linux is a feature of the Linux kernel and a number of utilities
+    with enhanced security functionality designed to add mandatory access controls to Linux.
+    The Security-enhanced Linux kernel contains new architectural components originally
+    developed to improve security of the Flask operating system. These architectural components
+    provide general support for the enforcement of many kinds of mandatory access control
+    policies, including those based on the concepts of Type Enforcement, Role-based Access
+    Control, and Multi-level Security. 
```

Comment:
This is the same rationale as in rule `package_policycoreutils_installed`.

Suggestion: 
This package is required to operate and manage an SELinux environment and its policies, it provides utilities such as semanage, audit2allow, audit2why, chcat and sandbox.